src/worker.cc

```diff
@@ -155,6 +155,7 @@ const std::unordered_map<unsigned, std::string> OSRM_ERRORS_CODES{
     {124, R"({"code":"InvalidOptions","message":"Options are invalid."})"},
     {125, R"({"code":"InvalidOptions","message":"Options are invalid."})"},
     {126, R"({"code":"InvalidOptions","message":"Options are invalid."})"},
+    {127, R"({"code":"InvalidOptions","message":"Options are invalid."})"},
```

Comment:
Can you add a more recostings-specific error message here?

Reply:
i think we should save that for another pr. at the moment, to remain osrm compatible we decided to, as much as we could, use messages that existed in that project. maybe its time to open up a discussion around whether that really makes sense anymore. i hope you are cool with doing that on another issue/PR though?